.github/workflows/dependency-review.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Dependency Review'
```

Comment:
should we apply this globally at sentry rather than adding it to each repository separately?

Reply:
Yes, yes we should. :) 
I'll work with Jeffrey to get this added as one of the global workflows and applied to the right repositories.